README.md

```diff
@@ -299,6 +299,7 @@ The following is a list of all the configuration properties the debugger accepts
 |`cwd`|The path to the project root folder|`string`|`${workspaceFolder}`|
 |`sourceMaps`|Whether to use JavaScript source maps to map the generated bundled code back to its original sources|`boolean`|`true`|
 |`sourceMapPathOverrides`|A set of mappings for rewriting the locations of source files from what the source map says, to their locations on disk. See [Debugging with TypeScript and Haul](#typescript-and-haul) for details|`object`|n/a|
+|`debuggingEnabled`|Whether to enable debug mode. If set to "false", an application will be launch without debugging|`boolean`|`true`|
```

Comment:
I suppose that it is better to change the name of this property to `enableDebug` since it represents some question to user, like `Do you want to enable debug?`.